CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-x.x.x Release notes (yyyy-MM-dd)
+0.13.1 Release notes (yyyy-MM-dd)
```

Comment:
You don't need to change it, but for future reference, the `set-version` script handles changing this version.